Atomic/backends/_ostree.py

```diff
@@ -128,6 +128,8 @@ def install(self, image, name, **kwargs):
         return self.syscontainers.install(image, name)
 
     def uninstall(self, iobject, name=None, **kwargs):
+        if name is not None:
+            raise ValueError("System Containers uninstall do not support deletion using container name, Please consider trying atomic containers delete command")
```

Comment:
What about "System containers do not support--name.  Please use atomic uninstall NAME" ?